### DISCUSSION TITLE
Teensy 4.1 + MCP23017

### DISCUSSION THREAD

**Discussion by wavelondon** [tttapa/Control-Surface#911] [Q&A]
Hi,

I'm really struggling to get a MCP23017 to work with a Teensy 4.1. When I run i2c_scanner.ino I get.......

I2C device found at address 0x20

But I can't get any of the "examples" to work. 

Here's the one I've tried...
Control-Surface-main/examples/1. MIDI Output/3. Rotary Encoders/MCP23017-RotaryEncoder-Interrupts/MCP23017-RotaryEncoder-Interrupts.ino
Control-Surface-main/examples/1. MIDI Output/3. Rotary Encoders/MCP23017-RotaryEncoder-No-Interrupts/MCP23017-RotaryEncoder-No-Interrupts.ino
Control-Surface-main/examples/Arduino-Helpers/Hardware/Extended Input & Output/3. In&Out/MCP23017/MCP23017.ino

I'm probably doing something really stupid, so any advice would be gratefully received. 

Thanks Ash

**Comment by tttapa** [comment]
Unfortunately, I don't have an MCP23017 handy right now. I'll see if I can get my hands on one next week.
